apps/prairielearn/src/middlewares/AuthzAccessMismatch.tsx

```diff
@@ -17,65 +35,65 @@ interface PermissionData extends PermissionMeta {
   authnValue: boolean | string;
 }
 
-const PermissionsMeta = [
+export const PERMISSIONS_META = [
   {
     label: 'Administrator',
     key: 'is_administrator',
     type: 'boolean',
   },
   {
-    label: 'Course preview',
+    label: 'Course role',
+    key: 'course_role',
+    type: 'string',
+  },
+  {
+    label: 'Course previewer',
     key: 'has_course_permission_preview',
     type: 'boolean',
   },
   {
-    label: 'Course view',
+    label: 'Course viewer',
```

Comment:
These changes were made to exactly match the permissions that folks will see in the "Staff" page.